docs/paid-features/activating-the-enterprise-edition.md

```diff
@@ -8,28 +8,50 @@ redirect_from:
-
 ## If you're running on Metabase Cloud
 
 If you've signed up for or upgraded to a Pro or Enterprise plan on Metabase Cloud, all of this will be taken care of for you.
 
-## Where to get a license
+## If you're self-hosting Metabase
+
+To use your Pro/Enterprise features, you’ll need to do two things:
+
+- Download Metabase Enterprise Edition
+- Enter your license.
 
 You can get a license by signing up for a free trial of the [Pro or Enterprise edition plans](https://www.metabase.com/pricing), both of which can be self-hosted or hosted on Metabase Cloud.
 
-## How to activate your token when self-hosting
+### Download the Enterprise edition
+
+- [Enterprise Docker image](https://hub.docker.com/r/metabase/metabase-enterprise/) at `metabase/metabase-enterprise:latest`. (RECOMMENDED)
+- [Enterprise JAR](https://downloads.metabase.com/enterprise/latest/metabase.jar).
+
+You'll also need to set up a dedicated [application database](../installation-and-operation/configuring-application-database.md) to store your Metabase data.
+
+### Enter your license
+
+There are two ways to enter your license when self-hosting Metabase:
 
-If you chose to host Metabase yourself, you'll get an email containing a unique license token. But to use it, you'll need to install the right JAR file.
+1. **When Metabase is running**: go to **Settings** > **Admin settings**, and click **License and Billing** in the lefthand sidebar. Paste in your license token under **License** and click **Activate**.
 
-You can either:
+OR
 
-- [Download the latest metabase-enterprise JAR](https://downloads.metabase.com/enterprise/latest/metabase.jar) (the filename is the same, irrespective of your plan), or
-- [Get the latest Docker image](https://hub.docker.com/r/metabase/metabase-enterprise/) at `metabase/metabase-enterprise:latest`.
+2. **Before you start Metabase**: you can also set the license token with the [`MB_PREMIUM_EMBEDDING_TOKEN` environment variable](https://www.metabase.com/docs/latest/configuring-metabase/environment-variables#mb_premium_embedding_token). This environment variable must be set _before_ you start your Metabase.
 
-Run Metabase as you would normally, then go to **Settings** > **Admin settings**, and click **License and Billing** in the lefthand sidebar. Paste in your license token under **License** and click **Activate**.
+## Upgrading from the open-source edition of Metabase
 
-## **Validating your token**
+If you've been running the open-source edition of Metabase, you'll need to change to the Enterprise Edition (that goes for both the Pro and Enterprise plans).
+
+Assuming you've been using a [production application database](../installation-and-operation/configuring-application-database.md), you can simply swap out the OSS Docker image or JAR for the Enterprise Edition.
+
+- [Enterprise Docker image](https://hub.docker.com/r/metabase/metabase-enterprise/) at `metabase/metabase-enterprise:latest`.
```

Comment:
I'm not sure about pointing directly to latest. In general (not necessarily in this article, but overall) we should suggest that people pin the versions instead of always pulling latest... 
Also, do the versions of OSS vs EE have to match? Can someone go from v50 OSS to v52 EE?